Makefile

```
@@ -131,13 +131,14 @@ dev:
 	@$(MAKE) install
 	pnpm exec vitepress dev
 
-.PHONY: full-refresh
-full-refresh: ## Refresh md files from external repos (.docforge/), post-process, then local preview
+.PHONY: hugo-refresh
+hugo-refresh: ## Refresh md files from external repos (.docforge/), post-process, then local preview
```

checkmake (0.3.2)

[warning] Target body for "hugo-refresh" exceeds allowed length of 5 lines (6).

(maxbodylength)